### PR TITLE
Mollie documentation

### PR DESCRIPTION
**Pull request by TarVK**
Documentation for Mollie, and some stripe updates for the new payments service.

### CODE REVIEW

**Comment by Borales** [comment]
@TarVK we have this PR https://github.com/deity-io/falcon-platform-docs/pull/49, could we merge them together?

File: docs/platform/integration/stripe.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Our integration includes:
 
 - `client/src/pages/shop/Checkout/components/payments/Stripe.js` (Client side component)
 - `@deity-io/falcon-stripe-plugin`(Client side plugin)
-- `@deity/falcon-payments/src/provider/stripe.js` (Server side provider)
+- `@deity/falcon-payments/packages/falcon-payments-stripe` (Server side provider)
```
Comment (Borales) [inline docs/platform/integration/stripe.md:31]:
```suggestion
- `@deity/falcon-payments-stripe` (Server side provider)
```

**Review by Borales — CHANGES_REQUESTED** [review]

File: docs/platform/integration/stripe.md (hunk 1 above)
Comment (TarVK) [reply docs/platform/integration/stripe.md:31]: Ah right, it's the package name, not location... my bad.

**Comment by TarVK** [comment]
> @TarVK we have this PR #49, could we merge them together?

Yes I think we should be able to merge this PR into that one.

**Review by Borales — APPROVED** [review]

**Pull request merged**